docs/cli-reference.md

```
@@ -1,99 +1,103 @@
+| `--verbose`      |        | Show detailed log output                                            |                     |
+
+### Info / listing options
+
+| Option                   | Short  | Description                                               |
+|--------------------------|--------|-----------------------------------------------------------|
+| `--list-languages`       | `-ll`  | Print all supported language codes                        |
+| `--list-models`          | `-lmm` | List TTS models available from Kokoro API                 |
+| `--list-voices`          | `-lv`  | List voices available from the selected TTS provider      |
+| `--list-tts-providers`   | `-ltp` | Show all TTS providers and their configuration status     |
+| `--create-voice-samples` | `-cvs` | Generate an M4B sample file with all available voices     |
+| `--max-samples`          | `-ms`  | Maximum number of voice samples to create                 | `5` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)